### PR TITLE
tag family member client details with location_id

### LINKED ISSUE #1399: Tag new clients with location Id
Currently when a client is registered, the record is not tagged with the location Id.

- [ ] When adding a client, tag it with the current Operational area as the location id

Part of the fix for https://github.com/opensrp/opensrp-server-web/issues/812

### PR DESCRIPTION
**Pull request by LZRS**
resolves #1399

### CODE REVIEW

**Comment by Rkareko** [comment]
update to client core version to [v4.3.3-SNAPSHOT](https://github.com/opensrp/opensrp-client-core/releases/tag/v4.3.3-SNAPSHOT)

**Comment by LZRS** [comment]
The build is failing from the `kujaku` library, seems like it's still pointing to bintray

File: opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public void testProcessFamilyRegistration() {
         assertNotNull(eventClient.getEvent());
         assertEquals("2e1b9bc9-c437-42a6-ac12-3566ca620d3f", eventClient.getEvent().getBaseEntityId());
         assertEquals(location.getId(), eventClient.getEvent().getLocationId());
+        assertEquals(eventClient.getEvent().getLocationId(), eventClient.getClient().getLocationId());
```
Comment (Rkareko) [inline opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java:84]: Is this correct?

File: opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java [hunk 2]
```diff
@@ -101,6 +102,7 @@ public void testProcessMemberRegistration() {
         assertNotNull(eventClient.getEvent());
         assertEquals("0dd853ae-be8d-4bfc-956d-2fb91eb687e5", eventClient.getEvent().getBaseEntityId());
         assertEquals(location.getId(), eventClient.getEvent().getLocationId());
+        assertEquals(eventClient.getEvent().getLocationId(), eventClient.getClient().getLocationId());
```
Comment (Rkareko) [inline opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java:105]: Assert this against `location.getId()`

File: opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java [hunk 3]
```diff
@@ -121,6 +123,7 @@ public void testProcessUpdateMemberRegistration() {
...
```
Comment (Rkareko) [inline opensrp-reveal/src/test/java/org/smartregister/reveal/model/FamilyProfileModelTest.java:126]: Assert this against `location.getId()`

**Review by Rkareko — APPROVED** [review]

**Pull request merged**